app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -166,39 +169,23 @@ export class Hamburger extends Component {
               }}
             />
           </div>
-
           <div className={cx('export-block')}>
             <div className={cx('export-label')}>
               <FormattedMessage id={'Hamburger.export'} defaultMessage={'Export:'} />
             </div>
             <div className={cx('export-buttons')}>
               <div className={cx('export-button')}>
-                <GhostButton
-                  tiny
-                  onClick={() => {
-                    onExportPDF(launch);
-                  }}
-                >
+                <GhostButton tiny onClick={() => this.onExportLaunch('pdf')}>
```

Comment:
I'd make a class method for every export type so we don't have to make new function on every render.
For example:
```
exportAsPDF = () => this.exportLaunch('pdf');

render() {
  //...
  <GhostButton tiny onClick={this.exportAsPDF}>
  // ...
}
```

Reply:
+1